projects/openttd_vs140.vcxproj.filters

```diff
@@ -28,78 +28,81 @@
     <Filter Include="MD5">
       <UniqueIdentifier>{c76ff9f1-1e62-46d8-8d55-000000000008}</UniqueIdentifier>
     </Filter>
-    <Filter Include="Script">
+    <Filter Include="Compat">
```

Comment:
Why have all of these filter/group names switched around... weird.

Reply:
generate script uses alphabetical order

Reply:
It is also a diff algorithm artifact. A better diff would simply show `Compat` being inserted after MD5 instead of all this single line shifting.

Reply:
I guess it's caused by all the UniqueIdentifier values getting renumbered.